components/base/InlineCTA.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import styled from "@emotion/styled";
+import arrowIcon from "../../public/images/arrow-right-orange.svg";
+import { GREY_2, WHITE_PRIMARY, PARAGRAPH_TEXT } from "../colors";
+import { BOX_SHADOW_5 } from "../shadowDepths";
+
+const BoxStyle = styled.div`
+  height: 78px;
```

Comment:
I think we could get away with removing `height`, `width` and the `@media` queries from a couple of these styled components and just adding a `max-width` here:

```diff
diff --git a/components/base/InlineCTA.tsx b/components/base/InlineCTA.tsx
index a982ec3..b314af4 100644
--- a/components/base/InlineCTA.tsx
+++ b/components/base/InlineCTA.tsx
@@ -5,8 +5,7 @@ import { GREY_2, WHITE_PRIMARY, PARAGRAPH_TEXT } from "../colors";
 import { BOX_SHADOW_5 } from "../shadowDepths";
 
 const BoxStyle = styled.div`
-  height: 78px;
-  width: 656px;
+  max-width: 656px;
   box-sizing: border-box;
   border: 1px solid ${GREY_2};
   border-radius: 5px;
@@ -17,28 +16,18 @@ const BoxStyle = styled.div`
   :hover {
     box-shadow: ${BOX_SHADOW_5};
   }
-  @media only screen and (max-width: 656px) {
-    height: auto;
-    width: auto;
-  }
 `;
 const IconStyle = styled.div`
   margin: 15px 33px 15px 15px;
 `;
 
 const TextStyle = styled.div`
-  height: 50px;
-  width: 506.84px;
   color: ${PARAGRAPH_TEXT};
   font-family: "Roboto", sans-serif;
   font-size: 15px;
   font-weight: 500;
   line-height: 25px;
   margin: 14px auto;
-  @media only screen and (max-width: 656px) {
-    height: auto;
-    width: auto;
-  }
 `;
 
 const ArrowStyle = styled.div`
```